### DISCUSSION TITLE
Is it possible to try this yet?

### DISCUSSION THREAD

**Discussion by AdnanuReeves** [Sil3ntWxlf/carpentersblocksreborn#1] [General]
Sorry I just stumbled upon this because I was looking for a carpenters blocks alternative for 1.16.4 and I found this and clicked the download link but it said no versions available. I'm assuming it's still in progress and no release has been made yet?

**Comment by Sil3ntWxlf** [comment]
Heya o/ I just saw this as I didn't get any notifications. The mod port is still in development as I am rewriting the mod entirely! When the mod is ready, I will push a release candidate for everyone to try out! Keep an eye out for the progress as we are pretty active with getting the mod created!

Currently we have a tonne of blocks being worked on and even brand new ones being added to CarpentersBlocks as well!
![image](https://user-images.githubusercontent.com/40921343/105570040-800fd500-5cfb-11eb-9e38-4f52db01ee2b.png)


The next things that we are implementing will be functionality and the custom voxel shapes.
